packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -185,6 +197,24 @@ const emit = defineEmits<{
       line-height: 1.5rem;
     }
 
+    $p: &;
+
+    &--ripple-animation {
+      &:hover:enabled {
+        --onyx-button-background-color: var(--onyx-button-background-hover-color);
+      }
+      #{$p}__label,
+      #{$p}__icon {
+        position: relative;
+        z-index: 1;
+        pointer-events: none;
+      }
+      #{$p}__loading {
+        pointer-events: none;
+        z-index: 1;
+      }
+    }
```

Comment:
Could you explain why this is needed? :)

Reply:
pointer-events: none is needed so the Ripple can recieve MouseEvents. I managed to remove the z-index.